tests/internal/aws_credentials_ec2.c

```diff
@@ -1,22 +1,23 @@
 /* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
 
-#include "aws_client_mock.h"
-#include "aws_client_mock.c"
-
 #include <fluent-bit.h>
 #include <fluent-bit/flb_sds.h>
+#include <fluent-bit/flb_time.h>
+#include <fluent-bit/flb_aws_util.h>
 #include <fluent-bit/flb_aws_credentials.h>
 #include <fluent-bit/aws/flb_aws_imds.h>
 #include <fluent-bit/flb_mem.h>
 #include <fluent-bit/flb_info.h>
 #include <fluent-bit/flb_http_client.h>
-#include <fluent-bit/flb_aws_util.h>
 #include <fluent-bit/flb_log.h>
 
 #include <monkey/mk_core.h>
 #include <string.h>
 #include <unistd.h>
 
+#include "aws_client_mock.h"
+#include "aws_client_mock.c"
```

Comment:
Why are there only import changes in this file? I think this was included by accident?